README.md

```diff
@@ -1,29 +1,60 @@
-# EC2-Minikube
-
-## Architecture
-
-![Architecture](images/architecture.drawio.png)
-
-## Pre-requisite
-- yq on your host instance, go [here](https://github.com/mikefarah/yq) to install
-
-## Usage
-
-### Provisioning the Infrastructure
-
-```bash
-terraform apply
-```
-
-### Connecting to the k8s cluster
-
-You can export the Kubeconfig file using this helper
-```bash
-$(terraform output -raw kubeconfig_command)
-```
-
-### Checking the connection
-
-```bash
-kubectl cluster-info
-```
+# EC2-Minikube
+
+## Architecture
+
+![Architecture](images/architecture.drawio.png)
+
+## Usage
+
+### Set up the aws Cli
+
+Use your access key and secret access key to authenticate
+
+```bash
+aws configure
+```
+
+### Add execution permission on "downoad_kubeconfig" script
+
+```bash
+sudo chmod +x ./scripts/download_kubeconfig.sh
+```
+
+### Install yq , yaml template engine .
+
+```bash
+sudo add-apt-repository ppa:rmescandon/yq

```

Comment:
You're assuming that people have ubuntu here 😂, just put a link to the repo and they would install it from there depending on their platform